Btchap/Config/BuildSettings.swift

```diff
@@ -119,13 +119,14 @@ final class BuildSettings: NSObject {
     static let applicationHelpUrlString = "https://tchap.beta.gouv.fr/faq"
 
     
-    // MARk: - Matrix permalinks
-    static let permalinkSupportedHosts: [String] = [
-        "beta.tchap.gouv.fr",
-        "www.beta.tchap.gouv.fr"
+    // MARK: - Matrix permalinks
+    // Hosts/Paths for URLs that will considered as valid permalinks. Those permalinks are opened within the app.
+    static let permalinkSupportedHosts: [String: [String]] = [
+        "matrix.to": ["/"],
+        "beta.tchap.gouv.fr": ["/"],
+        "www.beta.tchap.gouv.fr": ["/"],
     ]
-    static let permalinkPrefix = "https://www.beta.tchap.gouv.fr"
-    static let clientPermalinkBaseUrl: String? = nil
+    static let clientPermalinkBaseUrl: String? = "https://www.beta.tchap.gouv.fr"
```

Comment:
@phlniji can you please remove the "www." from the base url

Reply:
Fixed in last commit.